sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakePathAsyncClient.java

```diff
@@ -284,6 +287,10 @@ CpkInfo getCpkInfo() {
         return this.customerProvidedKey;
     }
 
+    boolean isTokenCredentialAuthenticated() {
```

Comment:
This method is not publicly accessible, correct?

Reply:
This is not a public method. It is protected within DataLakePathAsyncClient and cannot be accessed publicly.